src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php

```diff
@@ -33,10 +30,6 @@ protected function configure()
     {
         $this
             ->setName('cache:clear')
-            ->setDefinition(array(
-                new InputOption('no-warmup', '', InputOption::VALUE_NONE, 'Do not warm up the cache'),
```

Comment:
What is actually the strategy regarding those options? Removing them entirely is a BC break and `--no-warmup` was part of the upgrade path.
Should we just keep them and deprecated them to be removed in 5.0?